tests/conversion/view-v1-to-v2-test.js

```diff
@@ -0,0 +1,287 @@
+var expect = require('chai').expect
+var conversion = require('../../lib/conversion/view-v1-to-v2')
+var _ = require('lodash')
+
+var generateCellDefinitions = conversion.generateCellDefinitions
+// var generateCell = conversion.generateCells
+var convertCell = conversion.convertCell
+var viewV1toV2 = conversion.default
+
+describe('Bunsen view version 1 to view version 2 conversion', function () {
+  it('', function () {
+    expect(true).to.be.true
+  })
+})
+
+const CELL_DEF_TEST1 = {
+  description: 'converts a list of containers to cells',
+  inputs: [
+    [{
+      id: 'test',
+      rows: [
+        [
+          {
+            model: 'some-property',
+            container: 'someContainer'
+          }
+        ]
+      ]
+    }]
+  ],
+  expectedResult: {
+    test: {
+      children: [
+        {
+          model: 'some-property',
+          extends: 'someContainer'
+        }
+      ]
+    }
+  }
+}
+
+const CELL_TEST1 = {
+  description: 'converts array containers to cells',
+  inputs: [
+    {
+      model: 'some',
+      item: {
+        container: 'otherContainer'
+      }
+    }
+  ],
+  expectedResult: {
+    model: 'some',
+    arrayOptions: {
+      itemCell: {
+        extends: 'otherContainer'
+      }
+    }
+  }
+}
+
+const FULL_TEST1 = {
```

Comment:
Can we move these to separate files so this file doesn't become a monstrous beast. Maybe something along the lines of:

*tests/conversion/fixtures/array.js*

```js
export const v1 = {…}
export const v2 = {…}
```

*tests/conversion/view-v1-to-v2-test.js*
```js
import {
  v1 as arrayV1,
  v2 as arrayV2
} from './fixtures/array'
```

--

You could even go as far as adding an `index.js` in the fixtures file like so:

*tests/conversion/fixtures/index.js*

```js
import {
  v1 as arrayV1,
  v2 as arrayV2
} from './fixtures/array'

import {
  v1 as complexV1,
  v2 as complexV2
} from './fixtures/complex'

export default [
  {
    v1: arrayV1,
    v2: arrayV2
  },
  {
    v1: complexV1,
    v2: complexV2
  }
]
```

Then in the main test file you can do:

*tests/conversion/view-v1-to-v2-test.js*

```js
import tests from './fixtures'

…

tests.forEach((test) => {
 …
})
```